### PR TITLE
Adding files for mobile version

### PR DESCRIPTION
**Pull request by cmolinan**
### Added Changes for mobile version
Add "works" section
Add "about" section 
Update of README files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,29 +27,200 @@
...
+                    <img src="images/twitter.png" class="social" alt="twitter">
+                  </li>
+                  <li>
+                    <img src="images/medium.png" class="social" alt="Medium">
+                  </li>
+                </ul>
               </div>
         </section>
 
+        <section class="work-section">
```
Comment (tufoinnkuo10) [inline index.html:59]: - [ ] kindly maintain the white background in the work section as in the figma version

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-> Is the first milestone in the process of creating a portfolio website.
+> Is a new milestone in the process of creating a portfolio website.
 
 ## Built With
 
-- HTML / CSS (Flexbox)
+- HTML / CSS (Flexbox / GRID )
 
 ## Authors
```
Comment (tufoinnkuo10) [inline README.md:11]: - [ ] kindly add the authors name and other necessary links

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (tufoinnkuo10) [inline README.md:1]: - [ ] please add a brief description of what you did in the project

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @cmolinan 

Great job so far. There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅figma design followed

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
- [ optional ] please next time do not open a new pull request unless the old one has already been approved by a code reviewer.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by vickymarz — APPROVED** [review]
Hi @cmolinan , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**